packages/resource-deployment/scripts/install-parallel.sh

```diff
@@ -153,7 +149,6 @@ runInParallel parallelProcesses
 # Additionally, these should run sequentially because of interdependence.
 
 . "${0%/*}/batch-account-create.sh"
-. "${0%/*}/push-secrets-to-key-vault.sh"
```

Comment:
echo to batch-account-create comment - can we keep it at top level since hide it in batch-account-create not intuitive

Reply:
replied in the other comment.